create-table.md

```diff
@@ -1,22 +1,111 @@
 ---
 title: CREATE TABLE
 toc: false
+toc_not_nested: true
 ---
 
 The `CREATE TABLE` [statement](sql-statements.html) creates a new table in a database.
 
 <div id="toc"></div>
 
-## Synopsis
+## Syntax
 
 {% include sql/diagrams/create_table.html %}
 
 ## Required Privileges
 
 The user must have the `CREATE` [privilege](privileges.html) on the parent database. 
 
-## Parameters
 
-| Parameter | Description |
-|-----------|-------------|
-|  |  |
\ No newline at end of file
+## Semantics
+
+### IF NOT EXISTS
+
+Do not raise a **table already exists** error if a table of the same name in the database already exists. Note that the existence check is on the table name only, there is no guarantee that the existing table has the same columns, indexes, constraints, etc as the one that would have been created. 
+
+
+#### any_name
+
+The [name](data-definition.html#identifiers) (optionally schema-qualified) of the table to be created. Table names are required to be unique within a database.
+
+The `UPSERT` and `INSERT INTO ... ON CONFLICT` statements make use of a table called `EXCLUDED` to handle data conflicts during execution. It's therefore not recommended to use the name `EXCLUDED` for any of your tables.
+
+
+#### column_def
+
+An optional comma separated list of any [column names](data-definition.html#identifiers), their [data types](data-types.html), and any [column level constraints](data-definition.html#column-level-constraints). Column names have their own namespace within a table's definition so are required to be unique for the table but may have the same name as indexes or constraints, however this is not recommended. Any  Primary Key, Unique, and Check constraints that were originally defined at the column level are moved to the table level as part of the table's creation. Using the `SHOW CREATE TABLE` command will show these constraints at the table level.
+
+
+#### index_def
+
+An optional comma separated list of any [index definitions](data-definition.html#indexes). Index names have their own namespace within a table's definition so are required to be unique for the table but may have the same name as columns or constraints, however this is not recommended.
+
+
+#### table_constraint
+
+An optional comma separated list of any [table level constraints](data-definition.html#table-level-constraints). Constraint names have their own namespace within a table's definition so are required to be unique for the table but may have the same name as columns or indexes, however this is not recommended. 
+
+
+## Usage
```

Comment:
Since we're defining the parameters above, I don't think we need a "Usage" section. I'd rather have more detailed examples (see next comment). The notes about primary key and replication zones are helpful, so perhaps just add them to the general description at the top?